src/a11y/a11y.js

```diff
@@ -86,18 +92,32 @@ Object.assign(MediaElementPlayer.prototype, {
         return [...parentNode.childNodes].find(node => node.className.indexOf(className) > -1);
     },
 
+    /**
+     * Generates an HTML for an SVG icon. 
+     * @private
+     * @param {String} id
+     * @param {String} classPrefix
+     * @param {String} iconSpritePath
+     * @param {String} iconId
```

Comment:
What is the difference between `id` and `iconId`? We should explain the params in more detail in the param comment.

Reply:
Thanks